### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by maxsantos2606** [ultralytics/yolov5#10461] [Q&A]
I'm facing an error when running my project... could you help me?

`model = torch.hub.load(r'D:\Arduino\yolov5-master', 'custom', path=r'D:\Arduino\best.pt', source='local')

with mss() as sct:
    monitor = {"top": 220, "left": 640, "width": 640, "height":640}

    while(True):
        screenshot = np.array(sct.grab(monitor))
        results = model(screenshot, size=640)
        df= results.pandas().xyxy[0]
        try:
            xmin = int(df.iloc[0,0])
            ymin = int(df.iloc[0,1])
            xmax = int(df.iloc[0,2])
            ymax = int(df.iloc[0,3])
            
            cv2.rectangle(screenshot, (xmin, ymin), (xmax, ymax), (255,0,0), 2)
        except:
            print("",end="")

        cv2.imshow("frame", screenshot)
        if(cv2.waitKey(1) == ord('q')):
            cv2.destroyAllWindows()
            break`

my erro output: `File "C:\Users\Guto\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\nn\modules\module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
TypeError: forward() got an unexpected keyword argument 'size'`

**Comment by glenn-jocher** [comment]
@maxsantos2606 size argument works correctly with YOLOv5 PyTorch Hub models:

<img width="1433" alt="Screenshot 2022-12-10 at 13 16 23" src="https://user-images.githubusercontent.com/26833433/206875563-efa627fd-b2ed-479f-8001-7221a3992c39.png">

We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃
